pt/controller_1/pump.py

```diff
@@ -0,0 +1,79 @@
+import automationhat
+import time
+from influxdb_client import Point
+from store import InfluxDBStore
+
+from config import get_pump_id_by_pin, get_relay_by_pump_id
+from datetime import datetime
+
+
+def pump_water(sec: int, pump_pin: int):
```

Comment:
Please pass pump_id here. The required conversion from pump_pin to pump_id can be done in `comm.py` before calling this function.